### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by espertus**
These would have saved me and @jonalmeida time.

Instructions are here rather than in `shared-docs` at the recommendation of @mcomella: https://github.com/mozilla-mobile/shared-docs/pull/86

No tests were run because this does not change code.
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- [ ] **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- [ ] **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -246,6 +246,28 @@ _Sample apps using various components._
...
+
+If the environment variable `JAVA_HOME` is not defined, you will need to set it. If you would like to use the JDK installed by Android Studio, here's how to find it: 
+
+1. Open Android Studio.
+2. Select "Configure".
+3. Select "Default Project Structure". You should now see the Android JDK location. 
+4. Set the environment variable `JAVA_HOME` to the location. (How you set an environment variable depends on your OS.)
+5. Restart Android Studio.
+
+Once the environment variable is set, you can import the project into Android Studio with the default wizard options.
```
Comment (NotWoods) [inline README.md]: Should we including running gradle sync or any other steps after opening the project?

Comment (jonalmeida) [reply README.md]: That should be done automatically after the first clone of a-c and when we try to build.

File: README.md [hunk 2]
```diff
@@ -246,6 +246,28 @@ _Sample apps using various components._
...
 * [**Glean**](samples/glean) - An app demoing how to use the [**Glean**](components/service/glean/README.md) library to collect and send telemetry data.
 
+# Building #
+
+## Command line ##
+
+```
+$ git clone https://github.com/mozilla-mobile/android-components.git
+$ cd android-components
+$ ./gradlew
```
Comment (jonalmeida) [inline README.md]: Probably `./gradlew assemble`?

Comment (espertus) [reply README.md]: Done.

**Pull request merged**